rust/agama-lib/src/store.rs

```diff
@@ -184,33 +170,18 @@ impl Store {
         if let Some(user) = &settings.user {
             self.users.store(user).await?;
         }
-        // order is important here as network can be critical for connection
-        // to registration server and selecting product is important for rest
-        if let Some(product) = &settings.product {
-            self.product.store(product).await?;
-        }
-        // here detect if product is properly selected, so later it can be checked
-        let is_product_selected = self.detect_selected_product().await?;
-        if let Some(software) = &settings.software {
```

Comment:
how ordering will be done now? I expect that afterwards order of loading settings will be managed by manager, but how we achieve order now? Or we keep unattended broken meanwhile?

Reply:
Unattended is already broken. We are just removing the code piece by piece. All this logic will be in the new manager, behind the API.